get_started.md

```diff
@@ -2,42 +2,59 @@
 
 The Gaia Sandbox provides a containerized simulation environment that allows you to experiment with samples written using the Gaia Platform.
 
-This sample models a factory that contains the following:
+This sample models a security access control to allow or deny people entering an office campus. It contains:
 
-TODO
+- A headquarters (HQ) building
+- Rooms inside the HQ building
+- People, of varying security access levels
+    - John: an employee with access to all rooms
+    - Jane: a visitor with limited room access
+    - A stranger: they are allowed inside the HQ building!
+- Event schedules for people and rooms
 
 ## Overview of the Sandbox
 
 The Sandbox UI consists of the following elements.
 
-### Simulator window
+### User interface window
 
-TODO
+The right side of the screen shows the controls for the Access Control simulation. Each person in the **People** column has several icons: **Badge**, **Face**, **Parking**, and **WiFi**. These icons are all scanning options: for example, clicking on somebody's **Badge** icon simulates a "badge scan" into a building.
+
+Every person has a drop-down menu of the buildings/rooms they can scan into. Click on that box to choose a building/room, then click on one of the scan icons to choose a way to scan into that building/room. Every drop-down menu starts with only the **HQ Building** because you must scan into a building before you can scan into a room.
+
+You can also control time! Click the fast-forward icon by **Location at time** to set the time, which is important for visitors like Jane who cannot enter a room until a certain time.
 
 ### Ruleset, DDL, and Output tabs
 
 There are three tabs on the left. These are windows that show:
 
-- Ruleset: Displays the Declarative C++ code that handles TODO. You can modify the code in this tab and rebuild the sample to see how the changes affect the way that the sample runs. Rulesets encode the "business logic" that TODO
-- DDL: Displays the DDL code that describes the layout of the TODO, the state of the TODO, and events the factory produces.  In short, the DDL represents the data model for the app.
-- Output: Displays details when building/running your program.
+- **Ruleset:** Declarative C++ code that handles all the "business logic" of building and room access. You can edit the code in this tab and rebuild the sample to see how it affects the way that the sample runs.
+- **DDL:** A database schema describing the "data model" of everything in this Access Control scenario.
+- **Output:** Displays details while building/running your program.
```

Comment:
**Output**: Displays informational details when you are building or running the sample.